controllers/runner_manager_test.go

```diff
@@ -335,13 +335,13 @@ var _ = Describe("RunnerManager", func() {
 
 		By("checking metrics are not exposed")
 		MetricsShouldNotExist(metricsURL, "meows_runnerpool_replicas")
-		MetricsShouldNotExist(metricsURL, "meows_runner_online")
+		// MetricsShouldNotExist(metricsURL, "meows_runner_online")
```

Comment:
Why is it commented out here?
I have confirmed that test panics if I don't comment out this section, but I couldn't figure out why.

Reply:
Thank you!
It was commented out for debugging. And I forgot to turn it off.

Reply:
@kmdkuk 
Could you try `git pull --force`?
Before I did force-push yesterday, I faced a panic too. Now the bug is fixed.

Reply:
Thank you.
I've confirmed that the test works in my local environment!